drivers/ethernet/eth_mcux.c

```diff
@@ -601,9 +601,16 @@ static void eth_0_iface_init(struct net_if *iface)
 	context->iface = iface;
 }
 
-static struct net_if_api api_funcs_0 = {
-	.init	= eth_0_iface_init,
-	.send	= eth_tx,
+static enum eth_hw_caps eth_mcux_get_capabilities(struct device *dev)
+{
+	return 0;
+}
+
+static struct ethernet_api api_funcs_0 = {
```

Comment:
Shouldn't these structs be `const`?

Reply:
+1. Well, the original struct's weren't const, which may call for "make this a separate change", but I guess it's a pretty obvious change, to include into this commit.